apps/src/p5lab/AnimationTab/NewListItem.jsx

```diff
@@ -24,65 +24,80 @@ class NewListItem extends React.Component {
         type="button"
       >
         <div style={styles.wrapper}>
-          <div
-            style={[styles.dottedBorder, hovered && styles.dottedBorderHovered]}
-          >
-            <i className="fa fa-plus-circle" style={styles.addButton} />
+          <div style={[styles.border, hovered && styles.borderHovered]}>
+            <i
+              className="fa fa-plus-circle"
+              style={[styles.addButton, hovered && styles.borderHovered_add]}
+            />
+            <div
+              className="animation-name"
+              style={[
+                styles.animationName,
+                hovered && styles.borderHovered_animationName
+              ]}
+            >
+              {this.props.label}
+            </div>
           </div>
         </div>
-        <div className="animation-name" style={styles.animationName}>
-          {this.props.label}
-        </div>
       </button>
     );
   }
 }
-
+const polar_blue = '#0094CA';
```

Comment:
I found one other place where we use this color - in `AbuseExclamation.jsx`. Let's add this definition to our `colors.js` file and then reference that definition from here and `AbuseExclamation.jsx`.

Reply:
Investigated and resolved in discussion on Slack.

Reply:
Thanks Erin! We saw this color is not referenced in AbuseExclamation and color.js is not to be modified directly. So we will keep variable polar_blue as is in NewListItem.jsx.